tests/utils.rs

```diff
@@ -0,0 +1,270 @@
+use codec::Encode;
+use gear_lib::multitoken::{io::*, state::*};
+use gstd::{ActorId, String};
+use gtest::{Program, System};
```

Comment:
...and add `pub` to those! I think it's shorter and more convenient.
```suggestion
pub use gear_lib::multitoken::{io::*, state::*};
use gstd::{ActorId, String};
pub use gtest::{Program, System};
```